city_scrapers/settings/base.py

```diff
@@ -18,7 +18,7 @@
 USER_AGENT = "City Scrapers [development mode]. Learn more and say hello at https://www.citybureau.org/city-scrapers/"
 
 # Obey robots.txt rules
-ROBOTSTXT_OBEY = True
+ROBOTSTXT_OBEY = False
```

Comment:
Since you added `custom_settings` in the spider itself, you can change this back to `True`. We would like to avoid changing the settings.

Reply:
Yeah, this was from trying stuff out on scrapy shell, will revert